src/PKSim.Core/Services/BuildConfigurationTask.cs

```diff
@@ -56,6 +59,7 @@ public IBuildConfiguration CreateFor(Simulation simulation, bool shouldValidate,
 
          //STEP1: Create spatial structure
          buildConfiguration.SpatialStructure = _spatialStructureFactory.CreateFor(individual, simulation);
+         _parameterDefaultStateUpdater.UpdateDefaultFor(buildConfiguration.SpatialStructure);
```

Comment:
Code is not implemented yet in ParameterDEfaultStateUpdater. But calls are already available

Reply:
that's the comment here.